packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -14,6 +14,7 @@ import {
   getActionNameByEventName,
   RESIZE_EVENT,
 } from './utils/event';
+import { getSafeSize } from './utils/layout';
```

Comment:
Let's not create it in method. And better create correct isNumber() method in separate util file